docs/js/getting-started.md

```diff
@@ -20,53 +20,296 @@ import LicenseBadge from '../../src/sap/sdk-js/LicenseBadge'
+
+Copy the URL and paste it into the following command in your command line:
+
+```shell
+cf api https://api.cf.<region>.hana.ondemand.com
+cf login
+```
+
+### Before deploying
+
+- Make sure that your app listens to port 8080
+- Build your app if necessary
+
+### Deployment
+
+To deploy you app, run:
+```Shell
+npm run deploy
+```
+
+<!--This step consists of:
+
+- In productive environments:
+  - Transpile the application from TypeScript to JavaScript using the `ci-build` script
+  - Package the deployment using the `ci-package` script.
+  - Push to Cloud Foundry:
+```ts
+cf push
+```-->
+
+This command will use your local sources for transpiling, packaging and deployment, but will omit packaging your local `node_modules` as those can be system dependent. Dependencies will instead be installed automatically when deploying to `Cloud Foundry`.
+
+The Cloud Foundry CLI will automatically pick up the `manifest.yml` of the project when deploying your application. The file should look like this (where `<YOUR-APPLICATION-NAME>` is replaced by the name you specified when initializing the project):
+
+```YAML
+applications:
+  - name: <YOUR-APPLICATION-NAME>
+    path: deployment/
+    buildpacks:
+      - nodejs_buildpack
+    memory: 256M
+    command: npm run start:prod
+    random-route: true
+```
+
+- The specified `path` instructs Cloud Foundry to upload all the files from the `deployment/` folder.
+- The command specified under the `command` attribute tells the `buildpack` what command to issue to start the application.
+
+When everything works as expected, you should get output that looks something like this:
+
+```shell
+Waiting for app to start...
+
+name:              <YOUR-APPLICATION-NAME>
+requested state:   started
+routes:            <YOUR-APPLICATION-NAME>.cfapps.eu10.hana.ondemand.com
+last uploaded:     Thu 21 Mar 14:05:32 CET 2019
+stack:             cflinuxfs3
+buildpacks:        nodejs
+
+type:            web
+instances:       1/1
+memory usage:    256M
+start command:   node index.js
+     state     since                  cpu    memory        disk           details
+#0   running   2019-03-21T13:05:47Z   0.0%   16M of 256M   126.8M of 1G
+```
+
+The application will be running at the `routes` URL, you can also make sure that the application works correctly by running the start command, this command can be different than the one shown above.
+
+Should the application not work for whatever reason, you can call the following command to access the logs:
+
+```Shell
+cf logs <YOUR-APPLICATION-NAME> --recent
+```
+
+## Additional features
+
+For productive use, your app should be bound to a server, and also implement user authentication and authorization.
```

Comment:
```suggestion
For productive use, your app should be bound to a server and implement user authentication and authorization.
```

Reply:
I don't understand bound to a server. Can you explain?

Reply:
bound to a server -> linked to one or more databases?